pylabrobot/resources/corning_axygen/plates.py

```diff
@@ -0,0 +1,57 @@
+""" Corning Axygen plates """
+
+# pylint: disable=invalid-name
+
+from pylabrobot.resources.well import Well
+from pylabrobot.resources.itemized_resource import create_equally_spaced
+from pylabrobot.resources.plate import Plate
+
+from pylabrobot.resources.volume_functions import calculate_liquid_volume_container_2segments_square_vbottom
+
+
+def _compute_volume_from_height_Axy_24_DW_10ML(h: float):
+  if h > 42.1:
+    raise ValueError(f"Height {h} is too large for Cos_96_Vb")
+  return calculate_liquid_volume_container_2segments_square_vbottom(
+    x=17,
+    y=17,
+    z_pyramid=5,
+    z_cube=37,
+    liquid_height=h)
+
+
+#: Axy_24_DW_10ML
+def Axy_24_DW_10ML(name: str, with_lid: bool = False):
+  return Plate(
+    name=name,
+    size_x=127.0,
+    size_y=86.0,
+    size_z=44.24,
+    with_lid=with_lid,
+    model="Axy_24_DW_10ML",
+    lid_height=5,
+    items=create_equally_spaced(Well,
+      num_items_x=6,
+      num_items_y=4,
+      dx=10.3,
+      dy=5.8,
+      dz=2.24,
+      item_dx= 18,
+      item_dy=18,
+      size_x=17.0,
+      size_y=17.0,
+      size_z=42,
+      # bottom_type=WellBottomType.V,
+      # compute_volume_from_height=_compute_volume_from_height_Axy_24_DW_10ML,
+    ),
```

Comment:
Is this no longer needed?

Reply:
I initially commented it out because these arguments didn't work with the `Container` class I used to achieve the square rendering in the simulator.
If we modify the `Well` class with a e.g. `cross_section` = `square` or `round` argument then it would make sense to uncomment these arguments again - just done that.